hnn_core/viz.py

```diff
@@ -516,3 +518,59 @@ def plot_psd(dpl, *, fmin=0, fmax=None, tmin=None, tmax=None, layer='agg',
 
     plt_show(show)
     return ax.get_figure()
+
+
+def _plot_cell_morphology(ax, cell_type):
+    if cell_type == 'L2Pyr':
+        sec_pts, _, _, _, _ = _secs_L2Pyr()
+    elif cell_type == 'L5Pyr':
+        sec_pts, _, _, _, _ = _secs_L5Pyr()
+    else:
+        raise ValueError('Unrecognized cell type to plot')
+
+    for sec_pt in sec_pts.values():
+        xs = [pt[0] for pt in sec_pt]
+        ys = [pt[1] for pt in sec_pt]
+        zs = [pt[2] for pt in sec_pt]
+        ax.plot(xs, ys, zs, 'b.-')
+
```

Comment:
Would it be silly to just have a sphere plotted for basket cells? Just so that it is clear that they are point neurons.

Reply:
Are they actually point neurons or are they cylindrical neurons with one compartment?

Reply:
Cylindrical! And now I've added this.